src/main/scala/vsys/settings/WebhookEventRules.scala

```diff
@@ -54,7 +67,13 @@ object WithTxs extends RuleConfigReader {
   }
 }
 
-case class WithMintingTxs(value: Boolean) extends WebhookEventRules
+case class WithMintingTxs(value: Boolean) extends WebhookEventRules {
+  override def applyRule(height: Long, tx: ProcessedTransaction, accs: Set[Address]): Boolean = {
+    if (tx.transaction.transactionType == TransactionType.MintingTransaction)
+      if (value) true else false
+    else true
+  }
```

Comment:
This rule is applied to minting tx. When a processed transaction contains minting tx, it will always return true. If it is minting tx, it will apply the rule. I have updated this code to be neat.

Reply:
let me analyze the logic here
let A: `tx.transaction.transactionType == TransactionType.MintingTransaction`
B: value

|  A  | B |Return|
|---|---|---|
| true |  true |  true |
|  true |false   |false   |
|   false|  false |  true |
|false| true| ture|

The only case return false is A true B false, => !(A && !B) => !A || B, right?

Reply:
good catch~~